modules/logging-content-filter-drop-records.adoc

```diff
@@ -8,8 +8,14 @@
 
 When the `drop` filter is configured, the log collector evaluates log streams according to the filters before forwarding. The collector drops unwanted log records that match the specified configuration.
 
+[NOTE]
+====
+The `Fluentd` log collector does not support the `drop` filter. Please choose the `Vector` collector to make use of this feature.
+====
+
 .Prerequisites
 
+* You are using `Vector` as your log collector.
```

Comment:
`* Logging 5.9 configured to use the Vector collector.`

Reply:
LGTM